app/controllers/orders_controller.rb

```diff
@@ -0,0 +1,74 @@
+class OrdersController < ApplicationController
+  before_action :logged_in_user, only: %i(new create index)
```

Comment:
```suggestion
  before_action :logged_in_user
```
Chị nghix để vào dc controler ni thì bắt buộc phải login chứ ko có action nào là ngoại trừ cả nhỉ